server/app/services/export/enums/MultiOptionSelectionExportType.java

```diff
@@ -0,0 +1,17 @@
+package services.export.enums;
+
+/**
+ * the four options for a value are:
+ *
+ * <p>in the selected list and in the question definition and question is answered: SELECTED not in
+ * the selected list and in the question definition and question is answered: NOT_SELECTED not in
+ * the selected list and in the question definition and question is not answered: NOT_ANSWERED not
+ * in the selected list and not in the question definition: NOT_AN_OPTION_AT_PROGRAM_VERSION (or
+ * similar. This is both “retired” and “not yet an option at the time of this application”)
+ */
```

Comment:
**Nit (style)** Capitalization. Also can we format this as a list? We might need a line break or a `<p>` or a `<li>` so the formatter doesn't mess it up, though.

Alternatively, we could add them as line comments on each line in the enum